doc/website/blog/2022-08-11-announcing-versioned-migration-authoring.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Announcing v0.6.0 with Versioned Migration Authoring
+authors: a8m
+tags: [atlas, migrations, versioned]
+image: https://blog.ariga.io/uploads/images/posts/v0.5.0/atlas-lint.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+With the release of [v0.6.0](https://github.com/ariga/atlas/releases/tag/v0.6.0), we are
+happy to announce official support for a style of workflow for managing changes to database
+schemas that we have been experimenting with in the past months: _Versioned Migration Authoring_.
+
+### tl;dr
+
+* Atlas supports a declarative workflow (similar to Terraform) where users provide the 
+ desired database schema in a simple [data definition language](https://atlasgo.io/atlas-schema/sql-resources)
+ and Atlas calculates a plan to get a target database to that state. This workflow
+ is supported by the [`schema apply`](https://atlasgo.io/declarative/apply) command.
+* Many teams prefer a more imperative approach where each change to the database schema 
+ is checked-in to source control and reviewed during code-review. This type of workflow
+ is commonly called _versioned migrations_ and is supported by many established tools such
```

Comment:
Especially folks coming from Java Spring will know the term change based migrations. Maybe we add that term as well to decrease friction for readers from there?